lib/pbench/test/functional/agent/cli/commands/tools/test_tools_list.py

```diff
@@ -0,0 +1,8 @@
+import pytest
+
+
+def test_pbench_list_tools_help():
```

Comment:
While this is good to test the `--help`, can we add a test that just invokes `pbench-list-tools` with a default set of tools configured?  I am not sure the string formatting is going to do what we want for the logs.